lib/routes/account.js

```diff
@@ -476,7 +479,8 @@ module.exports = function (
                     )
                     .catch(
                       (err) => {
-                        if (err.errno === error.ERRNO.UNBLOCK_CODE_INVALID) {
+                        if (err.errno === error.ERRNO.INVALID_UNBLOCK_CODE) {
```

Comment:
This typo means that currently, invalid codes usage doesn't get tracked in the customs server :cry:. Adding the tests means we caught it now.